hwy/ops/x86_128-inl.h

```diff
@@ -1229,14 +1210,50 @@ HWY_API Vec128<int8_t, N> ShiftRight(const Vec128<int8_t, N> v) {
 template <typename T, size_t N, typename TI, size_t NI>
```

Comment:
This function will likely have to be moved after the definition of Load/Store to address the build issue:

```
In file included from /home/runner/work/highway/highway/hwy/highway.h:383:
/home/runner/work/highway/highway/hwy/ops/x86_128-inl.h:1230:3: error: use of undeclared identifier 'Store'; did you mean 'N_SSSE3::Store'?
  Store(Vec128<uint8_t>{BitCast(du8_bytes, bytes).raw}, du8_full, u8_bytes);
```